changelogs/fragments/173-argcomplete.yml

```
@@ -0,0 +1,5 @@
+minor_changes:
+  - "If you are using `argcomplete <https://pypi.org/project/argcomplete/>`__, you can now tab-complete ``antsibull-changelog`` command lines.
```

Technically, it was always possible to use tab completion manually with `eval "$(register-python-argcomplete ansible-changelog)"`.

```suggestion
  - "If you are using `argcomplete <https://pypi.org/project/argcomplete/>`__ global completion, you can now tab-complete ``antsibull-changelog`` command lines.
```